### PR TITLE
feat: use new config

### PR DESCRIPTION
**Pull request by holic**
taking another stab at #2470

### CODE REVIEW

File: packages/store-sync/src/zustand/logToTable.ts
Comment (holic) [inline packages/store-sync/src/zustand/logToTable.ts:1]: this isn't used, was moved up a directory but this never got removed

File: templates/react/packages/contracts/mud.config.ts [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
   tables: {
     Tasks: {
-      valueSchema: {
+      schema: {
+        id: "bytes32",
         createdAt: "uint256",
         completedAt: "uint256",
         description: "string",
       },
+      key: ["id"],
```
Comment (yonadaaa) [inline templates/react/packages/contracts/mud.config.ts:12]: OOC it is possible to use shorthand here? 

Or alternatively you could reduce the diff by keeping the `key` name

**Review by yonadaaa — APPROVED** [review]

File: templates/react/packages/contracts/mud.config.ts (hunk 1 above)
Comment (alvrs) [inline templates/react/packages/contracts/mud.config.ts:12]:
```suggestion
      id: "bytes32",
      createdAt: "uint256",
      completedAt: "uint256",
      description: "string",
```

Comment (alvrs) [reply templates/react/packages/contracts/mud.config.ts:12]: not blocking at all though

**Review by alvrs — APPROVED** [review]

**Pull request merged**